src/query/sql/src/planner/binder/copy.rs

```diff
@@ -194,15 +194,15 @@ impl<'a> Binder {
                 .await
             }
             (CopyUnit::Query(query), CopyUnit::UriLocation(uri_location)) => {
-                let ul = UriLocation {
+                let mut ul = UriLocation {
```

Comment:
Maybe we can keep `UriLocation` unchanged, and only introduce `Connection` struct during `parse_uri_location` so that we don't need to make `UriLocation` mutable.

But current logic LGTM too.

Reply:
emmm... `Connection` is part of `UriLocation`, split it out may be a little bit strange.